source/settings/config-file.txt

```diff
@@ -123,6 +123,10 @@ Settings
    * - trackUsageStatistics
      - Allow |compass-short| to send anonymous usage statistics.
 
+   * - :ref:`enableAIFeatures <enable-natural-language-querying>`
```

Comment:
[optional suggestion] Do we want to move this up so the settings are in alphabetical order? Totally understand if we want to keep it here at the bottom since the generative AI is in public preview.

Reply:
Good suggestion, I will put this in alpha order and confirm with the PM. Thanks!